plugins/backend/backstage-aws-backend/src/service/standaloneServer.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  createServiceBuilder,
+  loadBackendConfig,
+} from '@backstage/backend-common';
+import { Server } from 'http';
+import { Logger } from 'winston';
+import { createRouter } from './router';
+
+export interface ServerOptions {
+  port: number;
+  enableCors: boolean;
+  logger: Logger;
+}
+
+export async function startStandaloneServer(
+  options: ServerOptions,
+): Promise<Server> {
+  const logger = options.logger.child({ service: 'backstage-plugin-argo-cd-backend' });
```

Comment:
argo-cd?

Reply:
soz. I copied and pasted and badly covered my tracks.